docs/flink-writes.md

```diff
@@ -193,7 +193,7 @@ FlinkSink.builderFor(
 
 ### Branch Writes
 Writing to branches in Iceberg tables is also supported via the `toBranch` API in `FlinkSink`
-For more information on branches please refer to [branches](../../tables/branching).
+For more information on branches please refer to [branches](../branching).
```

Comment:
shouldn't this be `../tables/branching`? Same for the spark-writes.md

Reply:
I'm matching this URL https://iceberg.apache.org/docs/latest/branching/

Reply:
I think you actually need to match https://github.com/apache/iceberg-docs/blob/9c5cc5b241c177f153e5e2006eff1b8597c0c097/docs/content/branching-and-tagging.md?plain=1#L5